support-task/src/test/java/test/camunda/SupportTaskServiceImpl.java

```diff
@@ -30,8 +30,8 @@
         }
 
         // Add special variables
-        newVariables.put("ST:mainTaskInstanceId", mainTaskInstanceId);
-        newVariables.put("ST:supportTaskTypeKey", supportTaskTypeKey);
+        newVariables.put("mainTaskInstanceId", mainTaskInstanceId);
```

Comment:
We want to keep a suffix or other mechanism to distinguish between technical variables (metadata) and business variables (regular data)